README.adoc

```diff
@@ -258,7 +258,7 @@ $ just another-recipe
 This is another recipe.
 ```
 
-`just` prints each command to standard error before running it, which is why `echo 'This is a recipe!'` was printed. This is suppressed for lines starting with `@`, which is why `echo 'Another recipe.'` was not printed.
+Just prints each command to standard error before running it, which is why `echo 'This is a recipe!'` was printed. This is suppressed for lines starting with `@`, which is why `echo 'Another recipe.'` was not printed.
```

Comment:
Btw is "standard error" correct or should it be "standard output"?

Reply:
Yup, standard error is correct.